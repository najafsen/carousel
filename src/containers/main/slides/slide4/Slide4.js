import React from "react";
import "./Slide4.scss";

export const Slide4 = () => (
  <div className="slide-content">
    <div className="slide4">
      <h1>
        Or, for example, <span>Tivoli!</span>
      </h1>
    </div>
  </div>
);
