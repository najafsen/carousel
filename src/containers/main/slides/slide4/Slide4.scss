.slide4 {
  flex: 1;
  padding: 16px 80px;
  background-image: url("./img/tivoli.jpg");
  background-size: cover;
  text-align: right;

  h1 {
    font-size: 46px;

    span {
      color: #4282fd;
      text-transform: uppercase;
    }
  }
}
