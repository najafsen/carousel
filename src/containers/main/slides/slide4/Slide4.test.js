import React from "react";
import { render } from "@testing-library/react";
import { Slide4 } from "./Slide4";

describe("Slide4", () => {
  test("matches snapshot", () => {
    const { asFragment } = render(<Slide4 />);
    expect(asFragment()).toMatchSnapshot();
  });
});
