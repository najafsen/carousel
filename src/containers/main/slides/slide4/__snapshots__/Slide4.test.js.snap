// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Slide4 matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="slide-content"
  >
    <div
      class="slide4"
    >
      <h1>
        Or, for example, 
        <span>
          Tivoli!
        </span>
      </h1>
    </div>
  </div>
</DocumentFragment>
`;
